packages/examples/packages/preinstalled/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/preinstalled-example-snap",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
I wonder why we never released a version 1.0 of this example 😅

Reply:
Not sure 😅 We can do it for the next release, doesn't seem too important.